llvm/docs/ReleaseNotes.md

```diff
@@ -350,10 +350,16 @@ Changes to the Debug Info
 Changes to the LLVM tools
 ---------------------------------
 
+* llvm-objcopy now supports the following options for MachO:
+  `--globalize-symbol`, `--globalize-symbols`,
+  `--keep-global-symbol`, `-G`, `--keep-global-symbols`,
+  `--localize-symbol`, `-L`, `--localize-symbols`,
```

Comment:
I think you can omit `-G` and `-L` as they're just the short aliases for some of the other option names.

Reply:
Removed.